spec.html

```diff
@@ -22100,7 +22100,7 @@ <h1>
         </emu-alg>
         <emu-grammar>ForDeclaration : `using` ForBinding</emu-grammar>
         <emu-alg>
-          1. Assert: _environment_ is a declarative Environment Record.
+          1. Assert: _environment_ is a Declarative Environment Record.
```

Comment:
I suppose that's correct, though the section defining it uses lowercase-"d" for declarative in the definition.

Reply:
Are you looking at an old version of the spec? The current version has no occurrences of "declarative Environment Record". (Hasn't since PR #2640 was merged.)

Reply:
I must have been.